modules/frontend/search_response.go

```diff
@@ -139,3 +140,15 @@ func (r *searchResponse) result() *tempopb.SearchResponse {
 
 	return res
 }
+
+// coalesce combines two trace search metadata objects into one by taking the earliest start time and longest duration
+func coalesce(a, b *tempopb.TraceSearchMetadata) {
```

Comment:
looks like this function already exists: https://github.com/grafana/tempo/blob/main/tempodb/search/util.go#L55

Reply:
Changed 0245a3eb9